e2e/rbd.go

```diff
@@ -2119,17 +2111,66 @@ var _ = Describe("RBD", func() {
 			By("validate the functionality of controller", func() {
 				err := deleteResource(rbdExamplePath + "storageclass.yaml")
 				if err != nil {
-					e2elog.Failf("failed to delete storageclass with error %v", err)
+					e2elog.Failf("failed to delete storageclass : %v", err)
 				}
-				err = validateController(f, pvcPath, appPath, rbdExamplePath+"storageclass.yaml")
+				scParams := map[string]string{
+					"volumeNamePrefix": "test-",
+				}
+				err = validateController(f,
+					pvcPath, appPath, rbdExamplePath+"storageclass.yaml",
+					nil,
+					scParams)
 				if err != nil {
-					e2elog.Failf("failed to validate controller with error %v", err)
+					e2elog.Failf("failed to validate controller : %v", err)
```

Comment:
currently, I see `with error` in most of the places in E2E. Make a separate to change it across E2E to keep it consistent?

Reply:
https://github.com/ceph/ceph-csi/issues/2371

Opened an issue for this, we discuss and decide on how to standardise there.
Later on someone can keep it up to change current code, while others start following the new format.